frontend/src/metabase/query_builder/selectors.js

```diff
@@ -594,19 +594,22 @@ export const getIsObjectDetail = createSelector(
   (mode, isZoomingSingleRow) => isZoomingSingleRow || mode?.name() === "object",
 );
 
+export function isQuestionDirty(question, originalQuestion) {
```

Comment:
We extract the inline selector functions to be their own selector functions, so we can re-use them anywhere else.